packages/app/src/runner/SpecRunnerHeaderOpenMode.cy.tsx

```diff
@@ -208,4 +208,20 @@ describe('SpecRunnerHeaderOpenMode', { viewportHeight: 500 }, () => {
     cy.get('[data-cy="select-browser"] > button').should('be.disabled')
     cy.percySnapshot()
   })
+
+  it('opens and closes selector playground', () => {
+    cy.mountFragment(SpecRunnerHeaderFragmentDoc, {
+      render: (gqlVal) => {
+        return renderWithGql(gqlVal)
+      },
+    })
+
+    cy.get('[data-cy="playground-activator"]').click()
+    cy.get('#selector-playground').should('be.visible')
+
+    cy.percySnapshot()
```

Comment:
We document that we should have percy snapshots for "unique" states. I think how some tests are designed, they only produce one unique state so that's why there is only one snapshot, but for more complex tests that have a component go through multiple states than it makes sense to have multiple snapshots. I normally check the other tests before adding one just to verify that I'm not duplicating an existing snapshot.